doc/autoyast.md

```diff
@@ -170,29 +203,42 @@ Note that there are no separate classes for {Y2Storage::Planned::Disk} and
 
-{Y2Storage::AutoinstIssues} contains a set of possible issues (all of them
-are classes which inherit from {Y2Storage::AutoinstIssues::Issue}).
+{Y2Storage::AutoinstIssues} contains a set of possible issues (all of them are
+classes which inherit from {Y2Storage::AutoinstIssues::Issue}).
 
 [1]: https://doc.opensuse.org/projects/autoyast/#CreateProfile.General.storage "General section documentation"
 [2]: https://doc.opensuse.org/projects/autoyast/#CreateProfile.Partitioning "Partitioning documentation"
 [3]: https://github.com/yast/yast-autoinstallation/blob/20bf1d0ed6dca9d7bd194308db1baf76fe7312cd/src/clients/storage_auto.rb "storage_auto client"
-[4]: https://github.com/yast/yast-autoinstallation/blob/75af746a955be0d755e645da41061715329bcd7a/src/modules/AutoinstStorage.rb "AutoinstStorage module"
-[5]: https://github.com/yast/yast-autoinstallation/blob/20bf1d0ed6dca9d7bd194308db1baf76fe7312cd/src/modules/AutoinstStorage.rb#L78 "AutoinstStorage#import_general_settings"
-[6]: https://github.com/yast/yast-autoinstallation/blob/20bf1d0ed6dca9d7bd194308db1baf76fe7312cd/src/modules/AutoinstStorage.rb#L321 "AutoinstStorage#preprocessed_settings"
-[7]: https://github.com/yast/yast-autoinstallation/blob/20bf1d0ed6dca9d7bd194308db1baf76fe7312cd/src/modules/AutoinstStorage.rb#L255 "AutoinstStorage#build_proposal"
-[8]: https://github.com/yast/yast-autoinstallation/blob/20bf1d0ed6dca9d7bd194308db1baf76fe7312cd/src/modules/AutoinstStorage.rb#L276 "AutoinstStorage#valid_proposal?"
-[9]: https://github.com/yast/yast-autoinstallation/blob/20bf1d0ed6dca9d7bd194308db1baf76fe7312cd/src/modules/AutoinstStorage.rb#L300 "Present issues to the user"
-[10]: https://github.com/yast/yast-autoinstallation/blob/20bf1d0ed6dca9d7bd194308db1baf76fe7312cd/src/lib/autoinstall/storage_proposal.rb "StorageProposal"
+[4]: https://github.com/yast/yast-autoinstallation/blob/942413b8b54171ec3a79884c9be2138d11ba6803/src/modules/AutoinstStorage.rb "AutoinstStorage module"
+[5]: https://github.com/yast/yast-autoinstallation/blob/942413b8b54171ec3a79884c9be2138d11ba6803/src/modules/AutoinstStorage.rb#L75 "AutoinstStorage#import_general_settings"
+[6]: https://github.com/yast/yast-autoinstallation/blob/942413b8b54171ec3a79884c9be2138d11ba6803/src/modules/AutoinstStorage.rb#L316 "AutoinstStorage#preprocessed_settings"
+[7]: https://github.com/yast/yast-autoinstallation/blob/942413b8b54171ec3a79884c9be2138d11ba6803/src/modules/AutoinstStorage.rb#L251 "AutoinstStorage#build_proposal"
+[8]: https://github.com/yast/yast-autoinstallation/blob/942413b8b54171ec3a79884c9be2138d11ba6803/src/modules/AutoinstStorage.rb#L272 "AutoinstStorage#valid_proposal?"
+[9]: https://github.com/yast/yast-autoinstallation/blob/942413b8b54171ec3a79884c9be2138d11ba6803/src/modules/AutoinstStorage.rb#L296 "Present issues to the user"
+[10]: https://github.com/yast/yast-autoinstallation/blob/942413b8b54171ec3a79884c9be2138d11ba6803/src/lib/autoinstall/storage_proposal.rb "StorageProposal"
```

Comment:
I am not sure if it is better to link to fixed commit that can be really outdated or link to master, when lines may not fit.

Reply:
Yes, I am not sure either. However, I prefer to point to the right place even if it is outdated. If that's the case, you can investigate how the code has changed and try to find the right place again.